src/braket/aws/aws_device.py

```diff
@@ -137,23 +139,68 @@ def run(
         See Also:
             `braket.aws.aws_quantum_task.AwsQuantumTask.create()`
         """
-        if shots is None:
-            if "qpu" in self.arn:
-                shots = AwsDevice.DEFAULT_SHOTS_QPU
-            else:
-                shots = AwsDevice.DEFAULT_SHOTS_SIMULATOR
         return AwsQuantumTask.create(
             self._aws_session,
             self._arn,
             task_specification,
             s3_destination_folder,
-            shots,
+            shots if shots is not None else self._default_shots,
             poll_timeout_seconds=poll_timeout_seconds,
             poll_interval_seconds=poll_interval_seconds,
             *aws_quantum_task_args,
             **aws_quantum_task_kwargs,
         )
 
+    def run_batch(
+        self,
+        task_specifications: List[Union[Circuit, Problem]],
+        s3_destination_folder: AwsSession.S3DestinationFolder,
+        shots: Optional[int] = None,
+        max_parallel: int = AwsQuantumTaskBatch.MAX_PARALLEL_DEFAULT,
+        max_connections: int = AwsQuantumTaskBatch.MAX_CONNECTIONS_DEFAULT,
+        poll_interval_seconds: int = DEFAULT_RESULTS_POLL_INTERVAL,
```

Comment:
Should we add something for poll_timeout_seconds too?

Reply:
Those are in the `*args` and `**kwargs`; `poll_interval_seconds` is only here because it's actually used by AwsQuantumTaskBatch.